doc/usage/configuration.rst

```diff
@@ -393,9 +401,19 @@ General configuration
 
       Added ``i18n.inconsistent_references``
 
-    .. versionadded:: 7.1
+   .. versionadded:: 7.1
+
+      Added ``index`` warning type.
+
+   .. versionchanged:: 7.2
+
+      Make ``toc``, ``i18n`` and ``index`` warning types cover all relative
```

Comment:
I think this is not needed (only a changelog entry is needed). The `versionchanged` generally concerns changes at the core level of the object but you didn't change anything for `suppress_warnings` itself, right?

(Well I see that contributors actually hesitated a lot between aded/changed when introducing a new category haha).

As such, I would keep `versionchanged` for all `added ...` and remove the `Make ...`

Reply:
Yeah, you are right. Going to move them in next commit.

Introducing some new category can be a headache for newbies without the knowledge of project structure. :joy: